src/main/java/cz/codecamp/logger/loggers/PrintSteamLoggerLevelComparison.java

```diff
@@ -0,0 +1,29 @@
+package cz.codecamp.logger.loggers;
+
+import cz.codecamp.logger.FormatterInterface;
+import cz.codecamp.logger.LogLevelEnum;
+
+import java.io.PrintStream;
+
+/**
+ * Created by lenka.salacova on 10/15/2016.
+ */
+public class PrintSteamLoggerLevelComparison extends PrintStreamLogger {
+
+    private LogLevelEnum levelEnum;
+
+    public PrintSteamLoggerLevelComparison(PrintStream printStream, FormatterInterface formatter, LogLevelEnum levelEnum) {
+        super(printStream, formatter);
+        this.levelEnum = levelEnum;
+    }
+
+
+    @Override
+    public void log(LogLevelEnum level, String message) {
+        if (level.ordinal() >= levelEnum.ordinal()) {
```

Comment:
Tohle je pro `PrintStreamLogger`, šlo by to však ještě více zobecnit a použít např. [Decorator pattern](https://en.wikipedia.org/wiki/Decorator_pattern).